src/main/java/com/RegisterQuizApplication/Registration/QuizRepository.java

```diff
@@ -0,0 +1,19 @@
+package com.RegisterQuizApplication.Registration;
+
+import jakarta.persistence.criteria.CriteriaBuilder;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import java.util.List;
+
+public interface QuizRepository extends JpaRepository<Quiz,Integer> {
+    @Query(
```

Comment:
Dont need to use @Query unless its require.
Refer the link below
https://docs.spring.io/spring-data/jpa/reference/jpa/query-methods.html